launch/tier4_perception_launch/launch/object_recognition/detection/detection.launch.xml

```diff
@@ -21,6 +21,7 @@
   <arg name="camera_info7" default=""/>
   <arg name="image_number" default="1" description="choose image raw number(0-7)"/>
   <arg name="use_pointcloud_map" default="true" description="use pointcloud map in detection"/>
+  <arg name="use_validator" default="true" description="use obstacle_pointcloud based validator in detection"/>
```

Comment:
I don't want to add it to high level launch (detection.launch)

Reply:
Fixed.